libopenarc/Makefile.am

```diff
@@ -8,7 +8,7 @@ endif
 LDADD = ./libopenarc.la
 
 lib_LTLIBRARIES = libopenarc.la
-libopenarc_la_SOURCES = base64.c arc.c arc.h arc-canon.c arc-canon.h arc-dns.c arc-dns.h arc-internal.h arc-keys.c arc-keys.h arc-tables.c arc-tables.h arc-types.h arc-util.c arc-util.h
+libopenarc_la_SOURCES = base64.c base64.h arc.c arc.h arc-canon.c arc-canon.h arc-dns.c arc-dns.h arc-internal.h arc-keys.c arc-keys.h arc-tables.c arc-tables.h arc-types.h arc-util.c arc-util.h
```

Comment:
I'm not clear on what the distinction is supposed to be between openarc.conf.simple and openarc.conf.sample, or where the second one is supposed to come from. It's not currently present in the repository, so adding it to the Makefile won't fix the build.